spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/CredentialsPropertiesSetter.java

```diff
@@ -73,7 +71,8 @@ private static void setCredentialsEncodedKeyProperty(String encodedKey) {
         SqlCredentialFactory.class.getName());
   }
 
-  private static void setCredentialsFileProperty(Resource credentialsLocation, Log logger) {
+  static void setCredentialsFileProperty(Resource credentialsLocation)
+      throws IllegalArgumentException {
```

Comment:
`IllegalArgumentException` is a runtime exception; it does not need to be declared or propagated.